awscli/customizations/wizard/selectmenu.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from prompt_toolkit import Application
+from prompt_toolkit.utils import get_cwidth
+from prompt_toolkit.layout import Layout, FloatContainer, Float
+from prompt_toolkit.layout.controls import UIControl, UIContent
+from prompt_toolkit.layout.screen import Point
+from prompt_toolkit.layout.dimension import Dimension
+from prompt_toolkit.key_binding.key_bindings import KeyBindings
+from prompt_toolkit.layout.margins import ScrollbarMargin
+from prompt_toolkit.layout.containers import ScrollOffsets, Window
+
+
+def select_menu(items, display_format=None, max_height=10):
+    """ Presents a list of options and allows the user to select one.
+
+    This presents a static list of options and prompts the user to select one.
+    This is similar to a completion menu but is different in that it does not
+    allow a user to type and the returned value is always a member of the list.
+
+    :type items: list
+    :param list: The list of items to be selected from. If this list contains
+    elements that are not strings the display_format option must be specified.
+
+    :type display_format: Callable[[Any], str]
+    :param display_format: A callable that takes a single element from the
+    items list as input and returns a string used to represent the item in the
+    menu.
+
+    :type max_height: int
+    :param max_height: The max number of items to show in the list at a time.
+
+    :returns: The selected element from the items list.
+    """
+    app_bindings = KeyBindings()
+
+    @app_bindings.add('c-c')
```

Comment:
This causes issues for me on Python27. I'm getting the following output:
```
$ aws cli-dev wizard-dev --run-wizard file://role.yaml
Select type of trusted entity

Got 'c-c'
```

Reply:
Yeah I ran into that too the other day. cc @joguSD